src/SprykerEco/Shared/AmazonQuicksight/AmazonQuicksightConstants.php

```diff
@@ -0,0 +1,154 @@
+
+    /**
+     * Specification:
+     * - The AWS region that you use for the Amazon QuickSight account.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const AWS_REGION = 'AMAZON_QUICKSIGHT:AWS_REGION';
+
+    /**
+     * Specification:
+     * - AWS access key ID.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const AWS_CREDENTIALS_KEY = 'AMAZON_QUICKSIGHT:AWS_CREDENTIALS_KEY';
+
+    /**
+     * Specification:
+     * - AWS access key secret.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const AWS_CREDENTIALS_SECRET = 'AMAZON_QUICKSIGHT:AWS_CREDENTIALS_SECRET';
+
+    /**
+     * Specification:
+     * - AWS security token.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const AWS_CREDENTIALS_TOKEN = 'AMAZON_QUICKSIGHT:AWS_CREDENTIALS_TOKEN';
+
+    /**
+     * Specification:
+     * - The name of Quicksight namespace.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const AWS_QUICKSIGHT_NAMESPACE = 'AMAZON_QUICKSIGHT:AWS_QUICKSIGHT_NAMESPACE';
+
+    /**
+     * Specification:
+     * - The default data source username.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const DEFAULT_DATA_SOURCE_USERNAME = 'AMAZON_QUICKSIGHT:DEFAULT_DATA_SOURCE_USERNAME';
+
+    /**
+     * Specification:
+     * - The default data source password.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const DEFAULT_DATA_SOURCE_PASSWORD = 'AMAZON_QUICKSIGHT:DEFAULT_DATA_SOURCE_PASSWORD';
+
+    /**
+     * Specification:
+     * - The default data source database name.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const DEFAULT_DATA_SOURCE_DATABASE_NAME = 'AMAZON_QUICKSIGHT:DEFAULT_DATA_SOURCE_DATABASE_NAME';
+
+    /**
+     * Specification:
+     * - The default data source database port.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const DEFAULT_DATA_SOURCE_DATABASE_PORT = 'AMAZON_QUICKSIGHT:DEFAULT_DATA_SOURCE_DATABASE_PORT';
+
+    /**
+     * Specification:
+     * - The default data source database host.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const DEFAULT_DATA_SOURCE_DATABASE_HOST = 'AMAZON_QUICKSIGHT:DEFAULT_DATA_SOURCE_DATABASE_HOST';
+
+    /**
+     * Specification:
+     * - The default data source VPC connection ARN.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const DEFAULT_DATA_SOURCE_VPC_CONNECTION_ARN = 'AMAZON_QUICKSIGHT:DEFAULT_DATA_SOURCE_VPC_CONNECTION_ARN';
+
+    /**
+     * Specification:
+     * - The list of domains allowed for generating embed URLs.
+     *
+     * @api
+     *
+     * @var string
+     */
+    public const GENERATE_EMBED_URL_ALLOWED_DOMAINS = 'AMAZON_QUICKSIGHT:GENERATE_EMBED_URL_ALLOWED_DOMAINS';
+
+    /**
+     * Specification:
+     * - The role ARN used by `Aws\Sts\StsClient` to assume a role.
```

Comment:
Please mention that this role will be used for all future API interaction with the QS